src/js/modules/pagination-home.js

```diff
@@ -0,0 +1,17 @@
+import getBreeds from './GET-breeds-home';
+
+const pagination = async () => {
+// eslint-disable-next-line
+  const paginationContainer = document.querySelector('.pagination');
+  const dogBreeds = await getBreeds();
+  const numberOfItems = dogBreeds.length;
+  const numberPerPage = 6;
+  const currentPage = 1;
+  // eslint-disable-next-line
+  const numberOfPages = Math.ceil(numberOfItems/numberPerPage);
+  const trimStart = (currentPage - 1) * numberPerPage;
+  const trimEnd = trimStart + numberPerPage;
+  dogBreeds.slice(trimStart, trimEnd);
+};
```

Comment:
Both linter errors of this function come from `const` declarations that stay unused.
Are they necessary? Otherwise, please remove them and remove the linter disablers.